src/runtime/nitro/og-image/satori/plugins/imageSrc.ts

```diff
@@ -44,7 +44,7 @@ export default defineSatoriTransformer([
       if (imageBuffer)
         imageBuffer = Buffer.from(imageBuffer as ArrayBuffer)
       if (imageBuffer) {
-        node.props.src = toBase64Image(src, imageBuffer)
+        node.props.src = toBase64Image(imageBuffer)
```

Comment:
the first arg is still the file name so I think this would break?

Reply:
@harlan-zw I just modified the method signature 🤦‍♂️, I forgot to remove the filename which is no longer used